sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/AutoMLVertical.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.MachineLearning.Models
+{
+    /// <summary>
+    /// AutoML vertical class.
+    /// Base class for AutoML verticals - TableVertical/ImageVertical/NLPVertical
+    /// </summary>
+    public partial class AutoMLVertical
```

Comment:
`ML` or `Ml`?

Reply:
Considering that ML is a two-letter acrynm, it should be ML